tf_environment.yml

```diff
@@ -0,0 +1,8 @@
+name: pyvertical-dev-tf
+channels:
+  - tensorflow
+dependencies:
+  - python=3.8
+  - numpy >=1.16
+  - pandas >= 1.0.0
+  - tensorflow
```

Comment:
I would recommend installing `tensorflow` under pip within conda, because I don't think the conda binary is supported by the tensorflow team.

Also, we should pin tensorflow to either `>= 2.0.0` or `< 2.0.0` as `v2` is a significant API change

Reply:
Fixed.